### PR TITLE
completed es6-modules

### PR DESCRIPTION
**Pull request by BrianBozi**
Screen Shot 2021-07-07 at 12 52 50 PM

### CODE REVIEW

File: es6-modules/src/components/todo-app.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+// const { createElement, noop } = require('../lib');
+import createElement from '../lib/create-element';
```
Comment (scbowler) [inline es6-modules/src/components/todo-app.js]: Here you should be importing both `createElement` and `noop` from `lib/index.js` as named imports

```JAVASCRIPT
import { createElement, noop } from '../lib';
```

File: es6-modules/src/components/todo-form.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+// const { createElement } = require('../lib');
+import createElement from '../lib/create-element';
```
Comment (scbowler) [inline es6-modules/src/components/todo-form.js]: You should import this as a named import from `lib/index.js`

```JAVASCRIPT
import { createElement } from '../lib';
```

File: es6-modules/src/components/todo-list.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+// const { createElement } = require('../lib');
+import createElement from '../lib/create-element';
```
Comment (scbowler) [inline es6-modules/src/components/todo-list.js]: You should import this as a named import from `lib/index.js`

```JAVASCRIPT
import { createElement } from '../lib';
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Mostly looking good, but a few updates needed. Let me know if you have any questions

**Comment by BrianBozi** [comment]
made needed fixes

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**